Improved Team Year Selector

miguelaenlle opened the pull request:
10/20 Changes:
- Changed Select component font to Montserrat
- Increased Select component side padding 
- Added animation when dropdown opens/closes
- Changed dropdown option list border color to gray

10/25 Changes:

https://github.com/user-attachments/assets/44f146b0-bbd0-45d3-8f47-a0b713c8f006

- Added bar between 2025 and past years (only 2024 right now)
- Added mobile dropdown menu
- Implemented custom mobile dropdown menu
- Implemented click-off functionality on mobile and laptop/desktop (when the user clicks off the dropdown, it closes)

J164 commented on components/Select/Select.module.scss:
```diff
@@ -50,14 +50,19 @@
     text-align: center;
     border-radius: 10px;
-    border: solid 1px #000000;
+    box-shadow: rgba(99, 99, 99, 0.2) 0px -2px 8px 0px;
```
on mobile its a bit difficult to see the differentiation between the menu and content. Maybe the box shadow could be a bit darker/more obvious?

miguelaenlle replied:
I added this change 👍

J164 commented on components/Select/Select.module.scss:
```diff
@@ -67,4 +72,56 @@
+        max-height: 75%;
+        bottom: 0;
+        left: 0;
+
```
Could you increase the border radius on the top of the menu in mobile view? Right now the top of the menu is very flat and it might be more clear with something like 2rem.

miguelaenlle replied:
I added this change 👍

J164 reviewed (changes requested):
This is an amazing start! Would it be possible to add a close button (probably an 'x') to a top corner of the menu on mobile to make the user experience more clear? Great work so far!

miguelaenlle commented:
<img width="348" alt="image" src="https://github.com/user-attachments/assets/59d1ffb8-0b72-47b8-a2fa-1bed6742e0f2">

In the last commit, I added the following changes:
- Increased border radius and shadows on dropdown div on mobile
- Added X close button

J164 reviewed (changes requested):
This is awesome! One more thing: on desktop there seems to be a slight misalignment of the years
![image](https://github.com/user-attachments/assets/5d4024f1-bef6-4d57-bdd8-685230e8b4fa)

miguelaenlle commented:
![image](https://github.com/user-attachments/assets/6123a42c-616d-4702-86f5-d30995e23515)

- Fixed alignment of buttons in selector
- Slight adjustments to styling of buttons in the selector
- Fixed issue causing the dropdown to stay open when the user clicks the dropdown button multiple times
